nixos/modules/services/networking/whoogle-search.nix

```diff
@@ -0,0 +1,124 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.whoogle-search;
+  whoogle-search = pkgs.whoogle-search;
+  stateDir = "/var/lib/whoogle-search";
+in
+{
+  options = {
+    services.whoogle-search = {
+
+      enable = mkEnableOption "Whether to enable whoogle-search service.";
```

Comment:
No "Whether to enable" prefix. No trailing period. `mkEnableOption` already adds it.